### ISSUE TITLE
JG: No alt text for included graphics (converted from PDF to PNG)

### ISSUE THREAD

**Issue by hturner** [Abhi-1U/texor-rjarticle#18]
Comment from @ajrgodfrey

> There is no `alt` attribute for `<img>` tags when a PDF included with `\includegraphics` has been converted to an `<img>` tag including a PNG. In this case there should be a default alt text, e.g. `alt = "graphic without alt text"`.
> 
> Example: RJ-2009-015
>
> Also: is it necessary to use `<div class="center">` here, or could this be done with CSS on the image?
